manifests/sysmgmt.yaml

```
@@ -207,7 +207,7 @@ spec:
     namespace: services
   - name: csm-config
     source: csm-algol60
-    version: 1.33.0
+    version: 1.34.0
```

Pull in changes for CASMPET-7260
```suggestion
    version: 1.35.0
```